inspectit-ocelot-core/src/test/java/rocks/inspectit/ocelot/core/exporter/ExporterServiceIntegrationTestBase.java

```diff
@@ -57,7 +57,7 @@
 @Testcontainers(disabledWithoutDocker = true)
 abstract class ExporterServiceIntegrationTestBase extends SpringTestBase {
 
-    static final String COLLECTOR_IMAGE = "ghcr.io/open-telemetry/opentelemetry-java/otel-collector";
+    static final String COLLECTOR_IMAGE = "ghcr.io/open-telemetry/opentelemetry-java/otel-collector@sha256:d34519458388e55a3fce38a33e6bc424267c1f432927c09e932ba45f7575bd84";
```

Comment:
The issue was that not specifying any specific image leads to using the `latest` one which has been updated recently, which apparently broke our tests. Question is, does our implementation have to work with `latest` or is it enough to only work with the older one?